distributed/proxy/proxy_html.py

```diff
@@ -0,0 +1,132 @@
+from urllib import parse
```

Comment:
Let's go ahead and remove the unnecessary files for now.

Reply:
Ah, wait, this one *is* necessary, right?  Perhaps just the `Proxy` class at the bottom?

Reply:
If this module is just a single file then lets unpack it from a directory in order to avoid hierarchy.  Also, currently most of the dashboard stuff is in the `bokeh/` folder (which we should probably consider renaming `dashboard/`).  I recommend that we put this in there for now.